Tasks/VSMobileCenterUpload/package.json

```diff
@@ -20,9 +20,10 @@
     "vsts-task-lib": "0.9.20",
     "jszip" : "^3.1.2",
     "request": "2.72.0",
-    "glob": "^7.0.6"
+    "glob": "^7.0.6",
+    "nock": "9.0.2"
```

Comment:
@ericsciple 

Then I will add this to package.json at root of the repo. Going forward we can ask teams to add dev dependencies there. Does that look fine?